webpack.config.js

```diff
@@ -36,5 +38,23 @@ const config = {
       },
     ],
   },
+  node: {
+    __dirname: false,
+  },
+  plugins: [
+    new CopyWebpackPlugin({
+      patterns: [
+        {
+          from: path.join(require.resolve("mta-local"), "..", "bin"),
+          to: path.resolve(distPath, "bin"),
+        },
+      ],
+    }),
+    function (compiler) {
+      compiler.hooks.done.tap("ExecuteChmodOnBinMta", () => {
```

Comment:
Once you **validate()**, this code

1. unpacked_bin
2. creates an **empty** bin folder under`dist\bin`

- please write comments, what does it do
- this should be done during the "webpack" phase, not during "validate"